tests/integration/test_route_proxy.py

```
@@ -0,0 +1,228 @@
+        "model": "gpt-4o-mini",
+        "messages": [{"role": "user", "content": "ping"}],
+    }
+    try:
+        r = httpx.post(
+            f"{base_url}/v1/chat/completions",
+            json=payload,
+            headers={"Authorization": "Bearer irt-this-is-a-wrong-key"},
+            timeout=5,
+        )
+        if r.status_code == 401:
+            ok(f"POST /v1/chat/completions with bad key → 401 (correctly rejected)")
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)